app/src/main/java/com/abominableshrine/taptounlock/ShellUnlocker.java

```diff
@@ -0,0 +1,70 @@
+package com.abominableshrine.taptounlock;
+
+import android.util.Log;
+import android.view.KeyEvent;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+
+/**
+ * Lock and Unlock the device based on series of shell command executed as Root
+ */
+class ShellUnlocker {
+    private static boolean DEBUG = AppConstants.DEBUG;
+
+    /**
+     * This method locks the device again with a pattern or password.
+     *
+     * @see #unlock()
+     */
+    static void lock() {
```

Comment:
@psycholold I wonder if the name is correct, because the way I remember the shell command is that they just restore the password and key preferences without actually locking the device?